install.sh

```diff
@@ -93,12 +98,27 @@ install_driver() {
   fi
 }
 
+install_driver_dkms() {  
+    # Install Driver 
+    install -Dm 644 $(pwd)/dkms.conf /usr/src/maccel-${VERSION}/dkms.conf
+    
+    # Set name and version
+    sudo sed -e "s/@_PKGNAME@/maccel/" \
+         -e "s/@PKGVER@/${VERSION}/" \
+         -i "/usr/src/maccel-${VERSION}/dkms.conf"
+    
+    sudo cp -r $(pwd)/driver/* /usr/src/maccel-${VERSION}/
+
+    sudo dkms install "maccel/${VERSION}"
+
+    sudo modprobe maccel
```

Comment:
DKMS seems to already handle this on install in the `framework.conf` file.
```
       $modprobe_on_install
              Automatically load the built modules upon successful installation.
```

Reply:
Wow, I had no idea you could do that. I removed `modprobe` as suggested.
Is this setting something that should be mentioned in the README?

Reply:
I tested, It's not. I had to modprobe with it commented out. But it worked automatically with it enabled. 
I was also thinking that it would be a good improvement to mention adding the user to the maccel group in the README since the cli currently just crashes without it. And the only mention of it is in the install script (same for PKGBUILD).